### PR TITLE
Automate | LPS-197145 | Master

### PR DESCRIPTION
**Pull request by alanafergs**
Jira ticket: https://liferay.atlassian.net/browse/LPS-197145

Waiting for https://github.com/liferay-bpm-qa/liferay-portal/pull/1148/files be merged

### CODE REVIEW

**Comment by alanafergs** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
Hi @alanafergs, I think we should import the exported definitions and not import a new file. If we import a new file, the export and deletion steps of this test wouldn't really matter anymore. There are existing tests in portal where the exported file becomes a temporary file that you can still import in the same test. You can look at `LanguageOverrideExportImport#CanImportExportedPropertiesFile` as an example. Also, please separate your commits as needed rather than sending them all in one commit.

Let me know if you have questions. Thank you!

**Comment by alanafergs** [comment]
ci:reopen

**Comment by alanafergs** [comment]
ci:test:sf

**Comment by timpak** [comment]
https://github.com/liferay-bpm-qa/liferay-portal/pull/1148 should be ready sometime next week. I'll review this once it's merged.

**Comment by JhossephG** [comment]
> #1148 should be ready sometime next week. I'll review this once it's merged.

New PR was sent [here](https://github.com/liferay-bpm-qa/liferay-portal/pull/1308)

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase [hunk 1]
```diff
@@ -543,6 +545,114 @@ definition {
...
+			for (var objectName : list "A,B") {
+				ObjectAdmin.moveDefinitionFolder(
+					definitionName = "Custom Object ${objectName}",
+					folderLabel = "Custom Folder 2210");
+
+				Refresh();
+			}
+		}
+
+		task ("Then the object definitions are present and the relationship persists") {
```
Comment (JhossephG) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase:641]: Here you should use the macro instead the hard content from it @alanafergs. 
e.g. `ObjectAdmin.openObjectFolder(folderLabel = "Custom Folder");`

**Review by JhossephG — CHANGES_REQUESTED** [review]

**Pull request closed without merging**